aboutMe/apps.js

```diff
@@ -0,0 +1,55 @@
+//Q1
+const page=document.querySelector("page");
+style.fontfamily="Arial, sans-serif"
```

Comment:
Do not forget to add the variable name before adding the style.
```suggestion
page.style.fontfamily="Arial, sans-serif"
```